x-pack/plugins/security_solution/public/management/components/artifact_list_page/components/artifact_flyout.test.tsx

```diff
@@ -357,12 +359,19 @@ describe('When the flyout is opened in the ArtifactListPage component', () => {
         });
       });
 
-      expect(getLastFormComponentProps().item).toEqual({
+      const expectedProps = {
```

Comment:
This is unfortunate. Now that we have a generic Exceptions http mock, I should remove use of the trusted apps one here in this component and use instead the generic one.

Reply:
Yeah, I noticed that too. Let's do it in a new PR

Reply:
tracking this here elastic/security-team/issues/3685